RecoLocalCalo/HGCalRecProducers/plugins/HGCalRecHitWorkerSimple.h

```diff
@@ -47,6 +47,11 @@ class HGCalRecHitWorkerSimple : public HGCalRecHitWorkerBaseClass {
   std::vector<int> v_DB_reco_flags_;
   bool killDeadChannels_;
 
+  uint32_t GhostDetIdPosMin_;
+  uint32_t GhostDetIdPosMax_;
+  uint32_t GhostDetIdNegMin_;
+  uint32_t GhostDetIdNegMax_;
```

Comment:
start local variables and data members with lower case letter. Capitalization is reserved for class names.